src/common/password.cpp

```diff
@@ -134,7 +130,7 @@ namespace
     while (aPass.size() < tools::password_container::max_password_size)
     {
       int ch = getch();
-      if (EOF == ch)
+      if (EOF == ch || ch == 0x4)
```

Comment:
Please add a comment here explaining that `0x04 == EOT` (^D), or `#define EOT 0x04` somewhere.

Reply:
Technically EOF is wrong here too. `getchar` returns -1 (the EOF define) for any error. But anyway, added the EOT define.